src/graphs.py

```diff
@@ -152,8 +152,10 @@ def refresh(self):
     self.canvas.set_axis_properties()
     self.canvas.set_ticks()
     for item in reversed(self.datadict.values()):
-        if (item is not None) and not \
-                (self.preferences["hide_unselected"] and not item.selected):
-            self.canvas.plot(item)
+        if item is None \
+                or (self.preferences["other_hide_unselected"]
+                    and not item.selected):
+            continue
+        self.canvas.plot(item)
```

Comment:
I think these should eventually do the same. Don't really have a strong preference either way, they both work afaik and are readable to me.

Reply:
Main reason here was that the config key was too long to fit anymore, so made multiple attempts in rewriting ending up with this one.